less/_variables.less

```
@@ -201,7 +208,7 @@
 @box-shadow-lg:               0 1rem 3rem fade(@black, 17.5%);
 
 @component-active-color:      @white;
-@component-active-bg:         theme-color(primary);
+@component-active-bg:         @theme-colors[primary];
```

Let’s keep the usage of the functions here (and throughout this file) even though property accessors could be used. Having the Less be as close to the Sass version as possible makes it easy to quickly, visually compare the two. Thank you!
```suggestion
@component-active-bg:         @theme-color(primary);
```

---

It would be `theme-color(primary);` rather than `@theme-color(primary);`.  It'll take some time to be sure I restore them everywhere.

---

Also is this a change set that you'd like as part of #13 (the Less 3.9 version) as well?

---

No worries, I took care of this. Looks like they’re only used in this file, which was convenient. Yes, we’ll want to include this in the other PR.